app/modals/transaction/transaction-modal.tsx

```diff
@@ -57,10 +57,10 @@ import {
   modalStyle,
 } from './transaction-modal-layout';
 import { TxModalForm } from './steps/transaction-form';
-import { DecryptWalletForm } from './steps/decrypt-wallet-form';
-import { SignTxWithLedger } from './steps/sign-tx-with-ledger';
+import { SignTxWithLedger } from '../components/sign-tx-with-ledger';
```

Comment:
should be absolute

Reply:
In this instance, and a few others, they're relative paths as there's referencing a scoped components folder, not the main one. What is the pattern your suggesting, always absolute if it goes up a dir?